src/components/ErrorWrapper/index.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
```

Comment:
This isn't used

Reply:
Thanks it's a force of habit to always include that.